var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -64,3 +66,7 @@ def cmake_args(self):
                 'ON' if self.run_tests else 'OFF')
         ]
         return args
+
+    def install(self, spec, prefix):
+        super(PyPybind11, self).install(spec, prefix)
+        python('setup.py', 'install', '--prefix={0}'.format(prefix))        
```

Comment:
this would install the package twice... Why did you need to add it?

Reply:
Currently the folder .../gcc-5.4.0/py-pybind11-2.2.3-iecradcwti4qlm6qhrkmnh2bb4t2s62i/lib/python2.7/site-packages/ is empty since the installation is not complete. If a project tries to import pybind11 in the setup.py it cannot find this module even if loaded. Comparing to the build.sh from anaconda I added this missing part. One thing that is missing is to set the variable PYBIND11_USE_CMAKE=1. Do you know a way to do so?

Reply:
environment variable should be set now

Reply:
@toffer93 are you relying on `pybind11.get_include()`? If so, this is not yet properly set.
We probably have to provide that functionality upstream first, e.g. via a `PYBIND11_CMAKE_PREFIX` environment var during install with `PYBIND11_USE_CMAKE=1`.

Reply:
the package py-projectq uses `pybind11.get_include()`. But I do not understand why this functionality is missing in the current package setup.

Reply:
the problem is with `PYBIND11_USE_CMAKE=1` and installing in a prefix that is not the python prefix (which is ok, but a corner-case not yet covered by the pybind11 installer). Same issue: https://github.com/conda-forge/pybind11-feedstock/pull/32

We either have to patch [`pybind11/__init__.py`](https://github.com/pybind/pybind11/blob/v2.3.0/pybind11/__init__.py) on install with package managers and/or provide a fix upstream.

Reply:
the problem is with `PYBIND11_USE_CMAKE=1` and installing in a prefix that is not the python prefix (which is ok, but a corner-case not yet covered by the pybind11 installer). Same issue: https://github.com/conda-forge/pybind11-feedstock/pull/32

We either have to patch [`pybind11/__init__.py`](https://github.com/pybind/pybind11/blob/v2.3.0/pybind11/__init__.py) on install with package managers and/or provide a fix upstream.